sparkles/core.py

```diff
@@ -1351,6 +1353,25 @@ def check_include_exclude(self):
 
                     self.add_message("info", msg)
 
+    def check_t_ccd_acq(self):
+        """
+        Check that the fid lights are in the current search boxes by a check on t_ccd_acq.
+        These tolerances apply to observations after the 2022:294 safe mode.
+        """
+        if not self.is_ER and CxoTime(self.date).date > "2022:294":
+            if self.fids.t_ccd_acq <= -14.0:
+                self.add_message(
+                    "critical",
+                    f"Fid lights outside boxes (t_ccd_acq {self.fids.t_ccd_acq:.1f} <= -14.0)",
+                )
+                return
+            if self.fids.t_ccd_acq <= -13.5:
```

Comment:
This is not the best place for the early-return pattern because it does not reduce complexity. Instead:
```suggestion
            elif self.fids.t_ccd_acq <= -13.5:
```